mission_yaml/SEA56_M54.yml

```diff
@@ -0,0 +1,619 @@
+    standard_name: raw_chlorophyll_in_sea_water
+    units:        counts
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+
+  turbidity:
+    source:  FLNTU_NTU_SCALED
+    long_name:    turbidity
+    standard_name: turbidity_in_sea_water
+    units:        NTU
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_turbidity_meter
+    valid_min:    "0."
+    valid_max:    "25."
+    observation_type: "calculated"
+    sensitivity:      "0.01"
+
+  turbidity_raw:
+    source:       FLNTU_NTU_COUNT
+    long_name:    raw turbidity
+    standard_name: raw_turbidity_in_sea_water
+    units:        counts
+    coordinates:  time depth latitude longitude
+    instrument:    instrument_flourometer
+
+# oxygen:
+  oxygen_concentration:
+    source:       AROD_FT_DO
+    long_name:    oxygen concentration
+    standard_name: mole_concentration_of_dissolved_molecular_oxygen_in_sea_water
+    units:        mmol m-3
+    coordinates:   time depth latitude longitude
+    correct_oxygen: "True"
+    reference_salinity: "0"
+    instrument:    instrument_phosphorescence
+    valid_min:    "0."
+    valid_max:    "425."
+    observation_type: "calculated"
+    accuracy:      "2% or 2.0"
+    resolution:      "0.01"
+
+  temperature_oxygen:
+    source:  AROD_FT_TEMP
+    long_name:    oxygen sensor temperature
+    standard_name: temperature_of_sensor_for_oxygen_in_sea_water
+    units:        Celsius
+    coordinates:   time depth latitude longitude
+    instrument:    instrument_thermistor
+    valid_min:    "-3."
+    valid_max:    "45."
+    observation_type: "measured"
+    accuracy:      "0.01"
+    resolution:      "0.001"
+
+# methane:
+  methane_raw_concentration:
+    source:       METS_METHANE_VOLT
+    long_name:    methane raw concentration
+    standard_name: methane_raw_concentration
+    units:        volts
+    coordinates:   time depth latitude longitude
+    instrument:    Nan
+    valid_min:    "Nan"
+    valid_max:    "Nan"
+    observation_type: "calculated"
+    accuracy:      "Nan"
+    resolution:      "Nan"
+
+  methane_concentration:
+    source:       METS_METHANE_SCALED
+    long_name:    methane concentration
+    standard_name: methane_concentration
+    units:        mg m-3
+    coordinates:   time depth latitude longitude
+    instrument:    Nan
+    valid_min:    "Nan"
+    valid_max:    "Nan"
+    observation_type: "calculated"
+    accuracy:      "Nan"
+    resolution:      "Nan"
+
+  mets_raw_temperature:
+    source:       METS_TEMP_VOLT
+    long_name:    mets sensor raw temperature
+    standard_name: sensor_raw_temperature
+    units:        volts
+    coordinates:   time depth latitude longitude
+    instrument:    Nan
+    valid_min:    "Nan"
+    valid_max:    "Nan"
+    observation_type: "calculated"
+    accuracy:      "Nan"
+    resolution:      "Nan"
+
+  mets_temperature:
+    source:       METS_METHANE_SCALED
```

Comment:
Martin corrected this typo in methane. mets_temperature is from METS_TEMP_SCALED, not METS_METHANE_SCALED. METS_METHANE_SCALED is already used for the variable methane_concentration  You can see the PR here:

https://github.com/voto-ocean-knowledge/deployment-yaml/pull/115/files